### PR TITLE
740

### PR DESCRIPTION
**Pull request by sjoerdvanloon**

### CODE REVIEW

**Comment by sjoerdvanloon** [comment]
Prepped for 740

File: Changelog.md [hunk 1]
```diff
@@ -4,6 +4,42 @@ This page defines all the changes that were done in the API. There is a differen
...
+Added | **ANY** risks/control_test_reviews/{control_test_review_id}/actions | Manage actions on reviews
+Changed | **GET** risks/control_tests | Added parameter include_actions, include_reviews and card_ids
+Changed | **ANY** risks/controls/{control_id}/control_tests | Added support for hyperlinks on tests
+Changed | **POST** risks/controls/{control_id}/issues  | Added support for hyperlinks on issues
+Changed | **GET** risks/controls/filter/{filter_id} | Added include_custom_fields and include_custom_field_ids parameters to also include all or specific custom fields 
+Changed | **POST** risks/filter | Property 'main_risk_ids' added
+Added | **ANY** zenya_url_tokens | Added route to create tokens which can validate if the call was really done by your Zenya
+Deleted | **ANY** card_files/sections | **Breaking change** section management was never used from the api
+------------------------------
+Beta routes in 740 these routes are still subject to change.
```
Comment (LtDrak) [inline Changelog.md:33]: Beta routes; these are still subject to change

File: Changelog.md [hunk 2]
```diff
@@ -4,6 +4,42 @@ This page defines all the changes that were done in the API. There is a differen
...
+
+Change|Route|Remarks
+|--|--|--|
+Added | **ANY** data/ | Added API for module formely known as iData / Card files.
+Added  | **ANY** compliance/ | Added api for compliance
+Added | **ANY** search | Added API for AI search functionality
+Added | **ANY** support/example_questions | // Added api for AI questions  functionality
```
Comment (LtDrak) [inline Changelog.md:40]: // moet weg

File: Changelog.md [hunk 3]
```diff
@@ -4,6 +4,42 @@ This page defines all the changes that were done in the API. There is a differen
...
+Added | **ANY** finding_classifications and sub routes | Added API for finding classification
+Added | **GET** documents/{document_id}/contents/{language} | Get the contents of a document in a certain language
+Changed | **GET** hyperlinks | **Breaking change** Get a list of hyperlinks has been reimplemented to be faster and give more meta data
+Changed | **GET** risks | Added filter parameters main_risk_ids, added extension include_sub_risk_actions
+Changed | **POST** risks | Property 'is_main_risk'
+Changed | **PATCH** risks/{risk_id} | Property 'overridden_main_risk_properties' added
+Added | **ANY** risks/{risk_id}/issues | Added issue management
+Changed | **ANY** risks/risk_templates | Added support for framework limitations on the risk templates
+Changed | **ANY** risks/control_templates | Added support for framework limitations on the control templates
+Changed | **ANY** risks/control_templates | Added support for actions on control tests and control test reviews on the controle templates
```
Comment (LtDrak) [inline Changelog.md:22]: control templates

File: Changelog.md [hunk 4]
```diff
@@ -4,6 +4,42 @@ This page defines all the changes that were done in the API. There is a differen
...
+
+Change|Route|Remarks
+|--|--|--|
+Added | **ANY** audits and sub routes | Added API for audits
+Added | **ANY** findings and sub routes| Added API for findings
+Added | **ANY** finding_classifications and sub routes | Added API for finding classification
+Added | **GET** documents/{document_id}/contents/{language} | Get the contents of a document in a certain language
+Changed | **GET** hyperlinks | **Breaking change** Get a list of hyperlinks has been reimplemented to be faster and give more meta data
+Changed | **GET** risks | Added filter parameters main_risk_ids, added extension include_sub_risk_actions
+Changed | **POST** risks | Property 'is_main_risk'
```
Comment (LtDrak) [inline Changelog.md:17]: 'added'

File: Changelog.md [hunk 5]
```diff
@@ -4,6 +4,42 @@ This page defines all the changes that were done in the API. There is a differen
...
 [Pre 7.0.0 changes](ChangeLogs)
 
+## Changes in Zenya 7.4.0.0
+
+Change|Route|Remarks
+|--|--|--|
+Added | **ANY** audits and sub routes | Added API for audits
+Added | **ANY** findings and sub routes| Added API for findings
+Added | **ANY** finding_classifications and sub routes | Added API for finding classification
+Added | **GET** documents/{document_id}/contents/{language} | Get the contents of a document in a certain language
```
Comment (LtDrak) [inline Changelog.md:14]: Double check what this does

**Review by LtDrak — CHANGES_REQUESTED** [review]
Please check comments

File: Changelog.md (hunk 2 above)
Comment (sjoerdvanloon) [reply Changelog.md:40]: Fixes

**Review by LtDrak — APPROVED** [review]

**Pull request merged**